charts/dapr/charts/dapr_scheduler/templates/_helpers.tpl

```diff
@@ -33,7 +33,7 @@ Create initial cluster peer list dynamically based on replicaCount.
 {{- range $i, $e := until $replicaCount -}}
 {{- $instanceName := printf "dapr-scheduler-server-%d" $i -}}
 {{- $svcName := printf "%s.dapr-scheduler-server.%s.svc.cluster.local" $instanceName $namespace -}}
-{{- $peer := printf "%s=http://%s:%d" $instanceName $svcName (int $.Values.ports.etcdGRPCPeerPort) -}}
+{{- $peer := printf "%s=https://%s:%d" $instanceName $svcName (int $.Values.ports.etcdGRPCPeerPort) -}}
```

Comment:
Doesn't this need to be wrapped around a conditional on if mtls is enabled?

Reply:
@cicoyle no because we always do mtls on the control plane in Kubernetes environments